benchmarks/vector_db_compare/test_minima_treedb_runner.py

```
@@ -623,8 +988,23 @@ def test_artifact_uses_shared_segment_delta_resource_semantics(self) -> None:
         )
+        workload.source_commit = "a" * 40
+        workload.runner_sha256 = "b" * 64
+        workload.service_binary_sha256 = "c" * 64
+        workload.operation_timeout_seconds = 120
+        workload._phase_total_start = workload._phase_start = runner.time.monotonic_ns()
+        workload._phase_name = "initial_durable_load"
+        workload._phase_boundaries = []
+        workload._phase_attribution = None
+        workload._phase_resource_start = baseline
+        workload._phase_restart_old_end = None
+        workload.storage_path = Path("/tmp/data")
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/data"

(S108)